### PR TITLE
Update 01-what-why-and-how.md

### PR DESCRIPTION
**Pull request by jaredsmith**
This makes it clearer that SDP is *part* of the signaling, but isn't all of the signaling.  Also explains the SDP acronym.

### CODE REVIEW

**Comment by Sean-Der** [comment]
Nice! What do you think of making it more general. Instead of `media` rephrase it.

You can have calls with just a DataChannel and just want people to not just think media is all. We also have the bullet point list below that could be extended.

**Comment by jaredsmith** [comment]
Sounds like a good plan to me.

File: content/docs/01-what-why-and-how.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Each of these steps has dedicated chapters, but it is helpful to understand them
 
 When a WebRTC Agent starts it has no idea who it is going to communicate with and what they are going to communicate about. Signaling solves this issue! Signaling is used to bootstrap the call so that two WebRTC agents can start communicating.
 
-Signaling uses an existing protocol SDP. SDP is a plain-text protocol. Each SDP message is made up of key/value pairs and contains a list of 'media sections'. The SDP that the two WebRTC Agents exchange contains details like.
+An important part of signaling is to announce the types of media that may be exchanged between endpoints during a session.  This uses an existing protocol known as SDP -- the Session Description Protocol. SDP is a plain-text protocol. Each SDP message is made up of key/value pairs and contains a list of 'media sections'. The SDP that the two WebRTC Agents exchange contains details like.
```
Comment (michaelzg) [inline content/docs/01-what-why-and-how.md:48]: nit: `SDP (Session Description Protocol)` for consistency to when the other protocols are introduced in subsequent paragraphs.

**Comment by Sean-Der** [comment]
Merged with 0fcfac4

**Pull request closed without merging**